### PR TITLE
v8 10.5.218.7

### PR DESCRIPTION
**Pull request by chiefmikey**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

File: Formula/v8.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ class V8 < Formula
 
   resource "v8/third_party/icu" do
     url "https://chromium.googlesource.com/chromium/deps/icu.git",
-        revision: "1fd0dbea04448c3f73fe5cb7599f9472f0f107f1"
+        revision: "50ec7b3822a90d73aa761b21fc941b485a1cb9d6"
```
Comment (chiefmikey) [inline Formula/v8.rb:65]: @carlocab [This ref should actually exist](https://chromium.googlesource.com/chromium/deps/icu.git/+/50ec7b3822a90d73aa761b21fc941b485a1cb9d6) :)

**Comment by carlocab** [comment]
```
  clang++: error: unknown argument: '-femit-dwarf-unwind=no-compact-unwind'
```
Not sure why it's trying to use that on Catalina. OS detection code must be broken.

**Comment by cho-m** [comment]
> ```
>   clang++: error: unknown argument: '-femit-dwarf-unwind=no-compact-unwind'
> ```
> 
> Not sure why it's trying to use that on Catalina. OS detection code must be broken.

Looks intentional, e.g. https://chromium.googlesource.com/chromium/src/build.git/+/refs/heads/main/config/compiler/BUILD.gn#415
```gn
  # Apple compiler flags setup.
  # ---------------------------------
  if (is_apple) {
    # On Intel, clang emits both Apple's "compact unwind" information and
    # DWARF eh_frame unwind information by default, for compatibility reasons.
    # This flag limits emission of eh_frame information to functions
    # whose unwind information can't be expressed in the compact unwind format
    # (which in practice means almost everything gets only compact unwind
    # entries). This reduces object file size a bit and makes linking a bit
    # faster.
    # On arm64, this is already the default behavior.
    if (current_cpu == "x64") {
      asmflags += [ "-femit-dwarf-unwind=no-compact-unwind" ]
      cflags += [ "-femit-dwarf-unwind=no-compact-unwind" ]
    }
  }
```

May need to check if this is LLVM/Clang 15 option given how V8 upstream uses newer/unstable LLVM.

**Comment by carlocab** [comment]
Added in https://reviews.llvm.org/D122258, but seems to have been reverted and re-reverted after.

Not sure if it's made it to LLVM 15. Will have a look later.

**Comment by carlocab** [comment]
Yea, should be in LLVM 15 https://github.com/llvm/llvm-project/commit/d4bcb45db78dc7ca371224cb01bea8dcb14e0698.

**Comment by carlocab** [comment]
Alternatively, we can `inreplace` that flag out and remove the `inreplace` when we have LLVM 15. Though it's probably due soon, so we can probably just wait.

**Pull request closed without merging**